src/index.js

```diff
@@ -88,9 +103,9 @@ class Explosion extends React.PureComponent<Props, State> {
   };
 
   render() {
-    const { origin } = this.props;
+    const { origin, colors, fadeOut } = this.props;
     const { height, width } = Dimensions.get('window');
-
+    const colorsArray:Array<string> = colors ? colors : this.props.colors;
```

Comment:
space between const and type.
can you please use another wording for this ?

```
import defaultColors from '....';

[...]

const { colors: customColors } = this.props;
const colors: Array<string> = customColors || defaultColors;

Reply:
Btw, be careful, there is a bug there, you're not using the defaultColors in else case :)